### PR TITLE
Cherrypick cleanup and deprecation from #940 

### PR DESCRIPTION
**Pull request by Affie**

### CODE REVIEW

File: src/Deprecated.jl [hunk 1]
```diff
@@ -1061,3 +1062,169 @@ function addMsgFactors!(subfg::AbstractDFG,
...
+##------------------------------------------------------------------------------
+
+## TODO IS THIS DEPRECATED - only called in other code that looks to be deprecated
+
+# FIXME figure out if this can be consolidated and deprecated
+function dwnPrepOutMsg( fg::AbstractDFG,
+  cliq::TreeClique,
+  dwnMsgs::Array{LikelihoodMessage,1},
+  d::Dict{Symbol, T},
+  logger=ConsoleLogger()) where T
```
Comment (dehann) [inline src/Deprecated.jl:1079]: please add @warn("deprecated...") msgs.  Also I have been putting new stuff near the top of the file, just under "Deprecate after v0.16".

Comment (Affie) [reply src/Deprecated.jl:1079]: Ok, Can do warning, wanted it under a separate heading to first confirm before adding it to a version. Seems we just like opisites. Me adding at the bottom and you at the top :-)

Comment (dehann) [reply src/Deprecated.jl:1079]: its funny yeah

Comment (dehann) [reply src/Deprecated.jl:1079]: sorry also don't want to get in your way, i was more thinking that whatever is on its way does not really matter.  The few things i have under special heading is at the top so we dont forget about it.

Comment (Affie) [reply src/Deprecated.jl]: ok updated

**Comment by dehann** [comment]
not sure whats up with MASTER AND TAGS.  The same test failure happened twice in a row, but does not happen locally.  I created a new environment locally for MASTER and TAGS but tests fine here.  Might be another CSM quirk?

**Comment by dehann** [comment]
started it again for minor numerical hold up in testBasicGraphs

**Comment by dehann** [comment]
Change is very small on DFG?

https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/compare/v0.10.5...master

**Comment by dehann** [comment]
maybe some kind of precompile delay

**Comment by dehann** [comment]
yeah, 3rd time in a row failing at the same place on MASTER AND TAGS

```julia
FSM cliq4, iter=14 -- rmUpLikeliSaveSubFg_StateMachine, needdownmsg
FSM cliq4, iter=15 -- checkUpsolveFinished_StateMachine, needdownmsg
FSM cliq4, iter=16 -- trafficRedirectConsolidate459_StateMachine, needdownmsg
FSM cliq4, iter=17 -- waitChangeOnParentCondition_StateMachine, needdownmsg
FSM cliq3, iter=9 -- blockSiblingStatus_StateMachine, needdownmsg
FSM cliq3, iter=10 -- waitChangeOnParentCondition_StateMachine, needdownmsg
FSM cliq1, iter=7 -- trafficRedirectConsolidate459_StateMachine, null
FSM cliq1, iter=8 -- maybeNeedDwnMsg_StateMachine, null
FSM cliq1, iter=9 -- slowWhileInit_StateMachine, null
FSM cliq2, iter=18 -- rmUpLikeliSaveSubFg_StateMachine, upsolved
FSM cliq2, iter=19 -- checkUpsolveFinished_StateMachine, upsolved
FSM cliq2, iter=20 -- canCliqDownSolve_StateMachine, upsolved
FSM cliq2, iter=21 -- wipRedirect459Dwn_StateMachine, upsolved

No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.

Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received

The build has been terminated
```

**Comment by dehann** [comment]
im not sure what is going on here, but we have to move on.  I'm going to merge and we can figure out what is going on here

**Pull request merged**